pom.xml

```diff
@@ -173,11 +170,6 @@
         <version>${guava.version}</version>
         <scope>runtime</scope>
       </dependency>
-      <dependency>
-        <groupId>org.apache.httpcomponents</groupId>
-        <artifactId>httpclient</artifactId>
-        <version>${httpclient.version}</version>
-      </dependency>
```

Comment:
as long as it's the same version as spring uses it, we can keep it...

Reply:
I just reverted it. 😄